### ISSUE TITLE
Valid base 16: Not a base 16 number

### ISSUE THREAD

**Issue by MueR** [MikeMcl/bignumber.js#165]
From one day to the next, I'm getting a host of errors with this message and I can't for the life of me figure out what's going on. I've been using v6.0.0, tried upgrading to 7.0.0 (and 7.0.1), no difference.

Here's a trace: 
```
Error: [BigNumber Error] Not a base 16 number: 000000000190cb420dee880c5fa4a85cc572b411d5b6369693e4817a51f8df17
    at /Users/***/node_modules/bignumber.js/bignumber.js:1309:27
    at new BigNumber (/Users/***/node_modules/bignumber.js/bignumber.js:277:32)
    at BigNumber (/Users/***/node_modules/bignumber.js/bignumber.js:190:24)
    at Function.BigNumber.fromBuffer (/Users/***/src/util.mjs:41:12)
```
This is obviously a valid base16 number, coming in at 164877078785425560157898401589325588527433262895140429102633770775. I'm getting these on several numbers, not just this one. I can work around it by converting it to base10, but given that pretty much all my numbers are base16, that would be a massive amount of work.

If relevant, here's my config for bignumber.js:
```
BigNumber.config({
    ALPHABET: '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz',
    POW_PRECISION: 0,
})
```
I'm kind of at a loss here and would appreciate a hint.

**Comment by MueR** [comment]
Okay, once I remove the ALPHABET config, it works fine again. Which is weird, since all hex values get lowercased, but apparently not the alphabet?

**Comment by MikeMcl** [comment]
This is a bug that occurs if upper-case letters are used for hexadecimal (or other bases in the range 11-36) in a custom alphabet. 

I will probably publish the fixed version tomorrow.

Apologies for this bug and thanks for the report. The custom alphabet functionality lacked a proper test set and this is the result unfortunately.

**Comment by MikeMcl** [comment]
Fixed in *v7.0.3*.
